.github/workflows/build.yml

```diff
@@ -43,6 +43,8 @@ jobs:
             otp: '23'
           - elixir: '1.14'
             otp: '25'
+          - elixir: '1.16'
```

Comment:
Please see the comment on lines 15-26: https://github.com/ajvondrak/remote_ip/blob/0bbbb9660083c542db987986ad23a02ea9a2f5e0/.github/workflows/build.yml#L15-L26

Not that I do a great job of keeping the test matrix up to date with Elixir releases. 😅 Been 2 years (and 2 minor versions) since the last update, it seems. But whenever we _do_ update CI, I prefer to keep the same "shape" of the version combinations.

Reply:
The convention in the Elixir team (and the libraries that we collectively maintain) tends to be to test on the oldest-supported version and the latest-supported version, plus any *additional* version combination that we explicitly want to test — which tends to be none. This speeds up CI, pollutes less, and all of that 🙃 

What would you like the versions to test to be?